lib/tasks/enrollment_eligibility_report.rake

```diff
@@ -0,0 +1,52 @@
+desc "Run a report for enrollment eligibility"
+task :enrollment_eligibility_report => :environment do
```

Comment:
Might be helpful to specify that this is used for enrollment renewal eligibility, e.g., `enrollment_renewal_eligibility_report.rake`